draft-irtf-cfrg-frost.md

```diff
@@ -1011,7 +1011,7 @@ and 3) keep secret values confidential.
 
 It is assumed the dealer then sends one secret key share to each of the NUM_SIGNERS participants, along with `C`.
 After receiving their secret key share and `C` each participant MUST perform `vss_verify(secret_key_share_i, C)`.
-It is assumed that all participant have the same view of `C`.
+All participant MUST have the same view of `C`.
```

Comment:
If we're going to use normative language here, then we need to describe what happens or should happen when this isn't met. I'd revert this, too.

Reply:
After discussion with @dconnolly, we can just add "MUST abort if they do not have the same view of `C`"